packages/database/src/exp/Transaction.ts

```diff
@@ -88,7 +88,7 @@ export class TransactionResult {
  * transaction will be aborted and the data at this location will not be
  * modified.
  * @param options - An options object to configure transactions.
- * @returns A Promise that can optionally be used instead of the onComplete
+ * @returns A promise that can optionally be used instead of the `onComplete`
```

Comment:
Shouldn't it be \`Promise\` ?

Reply:
Either that, or just lowercase "promise."  With a cap it's a literal (correct?) and with lowercase it's just a plain old promise, which I think devs will understand.

Reply:
> * Uncapitalize "Promise" in several places since promises are standard parts of the language now? We don't capitalize `function` or `class`.

This was my rationale for uncapitalizing it. I think comments were written in the past when promises weren't standard and so we had written an actual custom implementation of them at some point?

Reply:
Interesting!   But lowercase seems to work well :)

Reply:
`Promise` is the name of the API and we write it in uppercase in code while we write `function`/`class` in lowercase. MDN uses capitalized `Promise` in their [doc]. so I think we should use literal `Promise`.

Reply:
A little hard to be sure I found all of them but I did as many as I could find.